### DISCUSSION TITLE
Can WebOne be used strictly as a HTTPS/TLS proxy?

### DISCUSSION THREAD

**Discussion by toontownlpz** [atauenis/webone#47] [Q&A] [answered]
I have a friend who is running a server that allows you to play Toontown Online using the original game client. Everything works as intended, other than the fact the launcher struggles to connect on Windows XP due to the modern TLS (I believe). My friend wants to host WebOne on his server strictly to allow users who want to play via Windows XP to connect. All OSes above XP can connect, as they support these modern security protocols by default.

Is this viable?
Are there any security issues with this?
Is it possible to prevent people directly connecting to his WebOne instance, so that it can be used strictly as a HTTPS/TLS proxy?

Apologies if I come across as dumb - I'm not that insightful on proxies and the like.

P.S. He has managed to setup WebOne on his server, however he does not want people to be able to connect to it to access every website under the sun - he just wants it so that people on Windows XP can access his website and play the game.

If there is another project that can accomplish this, I would greatly appreciate the link.

**Comment by atauenis** [comment, marked as answer]
No, WebOne allows only HTTP incoming connections. Currently it can't accept any TLS connections or any protocols except HTTP. For them, probably, ProxHTTPSproxy MII can be used to downgrade TLS, but only if the game is using Windows TLS stack.
